openapi-cli/src/test/java/io/ballerina/openapi/generators/service/ReturnTypeTests.java

```diff
@@ -283,4 +282,13 @@ public void singleResponseWithOutContent() throws IOException, BallerinaOpenApiE
         syntaxTree = ballerinaServiceGenerator.generateSyntaxTree();
         CommonTestFunctions.compareGeneratedSyntaxTreewithExpectedSyntaxTree("response/content_null.bal", syntaxTree);
     }
+
+    @Test
```

Comment:
Better to add a description for the test case as the method name seems not self-explanatory.